docs/faq/general/does-metabase-have-access-to-my-companys-data.md

```diff
@@ -0,0 +1,5 @@
+# Does Metabase have access to my company's data?
+
+If you run Metabase as a self-hosted instance, none of your data is collected or stored on Metabase servers.
+
+There is the option to opt-in to sharing anonymous usage data with us, which we use to make decisions about product improvements. This option is turned off by default, and you can read more about it [here](../../information-collection.md).
```

Comment:
technically speaking, there is no default as the admin always has to specify it on install. I think the default is "on" in that form. I would just strike the "turned off by default"